vtrace/platforms/win32.py

```diff
@@ -2210,3 +2223,116 @@ def getCacheSyms(self):
             ret.append( (addr-self.loadbase, size, name, stype) )
         return ret
 
+    def genSymbols(self, symopts=None, symtypes=None):
+        '''
+        Generator of symbols... does not cache symbols.  This is good for PDBs of unusual size (POUSes).
+        '''
```

Comment:
can't yield from the callback to the dbghelp.dll calls.  would be lovely if we could.  i like the idea of using a Queue.
however, remember the reason this is WIP.  this would be a ton easier if we simply had the caller hand in their own callback function.  that could update the workspace, or do whatever.

what do you think?  keep trudging toward the threaded/generator model?  or rework the calls to hand in a callback function?  although i'm not sure if the function handed in has to be a function, or if it could be a method with it's object attached.  if that, then all would work well (since the vw could be a field in the object for the callback).